ServerCore/Pages/Puzzles/Feedback.cshtml

```diff
@@ -8,11 +8,29 @@
     ViewData["PlayRoute"] = "../Puzzles/SubmitFeedback";
     ViewData["RoutingPuzzleId"] = ViewContext.RouteData.Values["puzzleId"];
     Layout = "_puzzleManagementLayout.cshtml";
-    var id = ViewContext.RouteData.Values["id"];
+
+    var sumFun = 0.0;
+    var sumDiff = 0.0;
+    var count = 0;
+    foreach (var item in Model.Feedbacks) {
+        sumFun += item.Fun;
+        sumDiff += item.Difficulty;
+        count++;
+    }
+    sumFun /= count;
+    sumDiff /= count;
 }
 
 <h2>@Html.DisplayFor(model => model.Puzzle.Name): Feedback</h2>
 
+<div>
+    <h3><b>Avg Fun Rating:</b> @sumFun</h3>
```

Comment:
Very small nit - maybe call this sumFunScore / sumDiffScore instead? It's pretty easy to mis-read these as sumFunction and sumDifference instead :P